wrapper.r2py

```diff
@@ -457,7 +457,11 @@ def _check_func_args(name, arg_definition, args):
       if "any" in expected_types:
         match = True
       else:
-        match = isinstance(args[index], expected_types)
+        # handling 'bool' seperately
+        if bool not in expected_types and isinstance(args[index], bool):
+          match = False
+        else:
+          match = isinstance(args[index], expected_types)
 
     # Only a single type
     else:
```

Comment:
Do you need to add code in here too?

Reply:
The relevant check here is `arg_type is expected_types`.
In our case, when `int` is passed as the only expected type and the function is passed either True/False, `arg_type` will be `bool`, and `expected_types` will be `int`.
`bool is int` will return False, and thus `match = False` in this scenario.